src/shared/settings.js

```diff
@@ -21,11 +21,11 @@ function assign(dest, src) {
  *
  * @param {Document|Element} document - The root element to search for
  *                                      <script> settings tags.
- * @param {string} settingsClass - The class name to match on <script> tags.
+ * @param {string} [settingsClass] - The class name to match on <script> tags.
  */
 function settings(document, settingsClass) {
   if (!settingsClass) {
-    settingsClass = 'js-hypothesis-settings';
+    settingsClass = 'js-hypothesis-config';
```

Comment:
Can we please just remove the configurability here? I can't think of any reason why we should be able to pass an alternate classname into this function.